### PR TITLE
Settings_import python "fixes" 

### PR DESCRIPTION
**Pull request by gavinelder**
Small items brought up by static analysis (SonarCloud) 

Remove Try / Except. 

Validate timezones.

### CODE REVIEW

File: docker/settings_import.py [hunk 1]
```diff
@@ -51,12 +44,7 @@
 
 # Read the preferred time zone from $TZ, use system locale or
 # set to 'America/New_York' if neither are set
-if getenv("TZ"):
-    if "/" in getenv("TZ"):
-        TIME_ZONE = getenv("TZ")
-    else:
-        TIME_ZONE = "America/New_York"
-elif getenv("TZ"):
+if getenv("TZ") and getenv("TZ") in pytz.all_timezones:
```
Comment (grahamgilbert) [inline docker/settings_import.py:47]: I don't think this error should be silently eaten - if someone chooses a wrong timezone, the application should error.

**Review by grahamgilbert — CHANGES_REQUESTED** [review]
See https://github.com/grahamgilbert/Crypt-Server/pull/118/files#r1116336655

File: docker/settings_import.py (hunk 1 above)
Comment (gavinelder) [reply docker/settings_import.py:47]: Happy to implement but didn't want to change the existing behaviour. 

Would you prefer warn or fatal ?

Comment (grahamgilbert) [reply docker/settings_import.py]: Fatal sounds good to me

**Pull request closed without merging**